SConstruct

```diff
@@ -330,8 +331,17 @@ qt_flags = [
   "-DQT_MESSAGELOGCONTEXT",
 ]
 qt_env['CXXFLAGS'] += qt_flags
-qt_env['LIBPATH'] += ['#selfdrive/ui', f"#third_party/maplibre-native-qt/{arch}/lib"]
-qt_env['RPATH'] += [Dir(f"#third_party/maplibre-native-qt/{arch}/lib").srcnode().abspath]
+
+maplibre_path = f"#third_party/maplibre-native-qt/{arch}/lib"
+if arch == "x86_64":
+  if UBUNTU_FOCAL:
```

Comment:
this is a bit janky. another option is to disable building the map stuff: https://github.com/commaai/openpilot/blob/149238845c1ed98dc5b86446c5c6d49e53b5e340/selfdrive/ui/SConscript#L12